docker-compose.yml

```diff
@@ -5,15 +5,19 @@ services:
     command: server
     deploy:
       replicas: 1
+      placement:
+        constraints: [node.role == manager]
```

Comment:
I think I can only get the official list of nodes in a swarm from the manager node -- that's why I constrained it